_meta/beat.yml

```diff
@@ -315,7 +315,7 @@ output.elasticsearch:
   # Scheme and port can be left out and will be set to the default (http and 9200)
   # In case you specify and additional path, the scheme is required: http://localhost:9200/path
   # IPv6 addresses should always be defined as: https://[2001:db8::1]:9200
-  hosts: ["localhost:9200"]
+  hosts: ["{{ .elasticsearch_hostport }}"]
```

Comment:
doing the string join with commas in the template was a pain that we can postpone until we really need this to be a list